### ISSUE TITLE
Can't switch tabs in TabbedInterface in image_classifier_interface_load demo

### ISSUE THREAD

**Issue by freddyaboulton** [gradio-app/gradio#2022]
### Describe the bug

1. Click on an example in the first tab and hit `Submit`
2. Go to the second tab and click on an example and hit `Submit`
3. Go back to the first tab

You will see both interfaces displayed in the same page (bug) and won't be able to switch tabs. See image below.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

```python
import gradio as gr
import pathlib

current_dir = pathlib.Path(__file__).parent

images = [str(current_dir / "cheetah1.jpeg"), str(current_dir / "cheetah1.jpg"), str(current_dir / "lion.jpg")]


img_classifier = gr.Interface.load(
    "models/google/vit-base-patch16-224", examples=images, cache_examples=False
)


def func(img, text):
    return img_classifier(img), text


using_img_classifier_as_function = gr.Interface(
    func,
    [gr.Image(type="filepath"), "text"],
    ["label", "text"],
    examples=[
        [str(current_dir / "cheetah1.jpeg"), None],
        [str(current_dir / "cheetah1.jpg"), "cheetah"],
        [str(current_dir / "lion.jpg"), "lion"],
    ],
    cache_examples=False,
)
demo = gr.TabbedInterface([using_img_classifier_as_function, img_classifier])

if __name__ == "__main__":
    demo.launch()
```

### Screenshot

![image](https://user-images.githubusercontent.com/41651716/184967864-ccd9ca8f-ebe9-4814-b5ef-e06c9e5f6efe.png)

![image_classifier_interface_load](https://user-images.githubusercontent.com/41651716/185163277-824caf81-e9af-47ac-bb26-df51d3cbc871.gif)

### Logs

```shell
-
```

### System Info

```shell
-
```

### Severity

blocking upgrade to latest version

**Comment by freddyaboulton** [comment]
Doesn't seem to impact the other TabbedInterface demo (sst_or_tts)

**Comment by abidlabs** [comment]
It looks like this is fixed now in `gradio==3.4` @freddyaboulton. Can you confirm?

**Comment by freddyaboulton** [comment]
It's fixed! Shout out @pngwn who probably did this in the sketching pr
